sass/layout/_sidebar-left.scss

```diff
@@ -599,26 +599,24 @@
         .SidebarMenu:not(.menuOpen) {
             display: none;
         }
+    }
 
-        &.draggingOverHeader:hover {
-            //background: radial-gradient(60.83% 3421.87% at 50% 50%, rgba(255, 255, 255, 0.4) 0%, rgba(255, 255, 255, 0) 100%);
-            box-shadow: inset -1px 0 0 rgba(var(--sidebar-text-rgb), 0.60), inset 0 -1px 0 rgba(var(--sidebar-text-rgb), 0.6), inset 1px 0 0 rgba(var(--sidebar-text-rgb), 0.6), inset 0 1px 0 rgba(var(--sidebar-text-rgb), 0.6);
-            border-radius: 4px;
+    .SidebarChannelGroup:not(.dmCategory) .draggingOver .SidebarChannelGroupHeader:hover {
```

Comment:
The previous code applied the `draggingOverHeader` class only sometimes depending on the type of the category. Since the UNREAD CHANNELS header doesn't have a category, I chose to always apply the class but add an additional class to the category to identify the DMs and let the CSS determine what styles DMs need vs other categories.